Changelog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 20230825 - New Endpoints
+
+### Request parameters and response schema has changed for endpoint "/v1/vouchers/{code}/validate"
```

Comment:
Changed? New types were added (like `ValidationsValidateVoucherParams`, `CustomerRequest`  ) that replaced the old ones as far as I see

Reply:
so it would be great to describe those new types in changelog IMHO